index.js

```diff
@@ -164,6 +164,9 @@ function SpotifyResolve(createOpts) {
       if (error) {
         done(error);
       }
+      else if (response.statusCode === 401) {
+        done(new Error(response.statusCode), response);
```

Comment:
Thanks for pointing me in the right direction! Didn't know how I missed that doc... should have dig deeper!